components/insight/build/dist.xml

```diff
@@ -450,19 +450,19 @@
     <property name="dist.zip.prefix.osx"
               value="${dist.bundle.name}-${dist.bundle.version}-mac"/>
 
-    <property name="dist.zip.prefix.osx.107"
-              value="${dist.zip.prefix.osx}_Java7+"/>
-    <zip destfile="${dist.dir}/${dist.zip.prefix.osx.107}.zip">
-      <zipfileset prefix="${dist.zip.prefix.osx.107}" dir="${dist.dir}"
+    <property name="dist.zip.prefix.osx"
+              value="${dist.zip.prefix.osx}"/>
+    <zip destfile="${dist.dir}/${dist.zip.prefix.osx}.zip">
+      <zipfileset prefix="${dist.zip.prefix.osx}" dir="${dist.dir}"
                includes="OMERO.insight.app/**"
                excludes="**/${distInsight.name}"/>
       <zipfileset file="${dist.dir}/OMERO.insight.app/Contents/MacOS/${distInsight.name}"
-                  fullpath="${dist.zip.prefix.osx.107}/OMERO.insight.app/Contents/MacOS/${distInsight.name}"
+                  fullpath="${dist.zip.prefix.osx}/OMERO.insight.app/Contents/MacOS/${distInsight.name}"
                   filemode="775"/>
-      <zipfileset prefix="${dist.zip.prefix.osx.107}"
+      <zipfileset prefix="${dist.zip.prefix.osx}"
                   file="${dist.osx.installfile}"/>
       <zipfileset file="${base.licensefile}"
-                  fullpath="${dist.zip.prefix.osx.107}/LICENSE"/>
+                  fullpath="${dist.zip.prefix.osx}/LICENSE"/>
     </zip>
 
     <delete dir="${dist.dir}/OMERO.insight.app"/>
```

Comment:
This whole block using the `jarbundler` can likely be removed either on this PR on on a follow-up one since it was specific to the Java6 bundle.